pkg/client/env.go

```diff
@@ -26,3 +27,20 @@ func (c Client) EnvByID(id string) (*types.Response, error) {
 
 	return types.UnmarshalEnv(body)
 }
+
+// AllEnvironmentUUIDs tries to fetch all environment by UUIDs in an org.
+func (c Client) AllEnvironmentUUIDs() (*types.UUIDResponse, error) {
+	params := make(map[string]string)
+	if c.Org != "" {
+		params["Org"] = c.Org
+	}
+
+	body, err := c.Requester.Do(http.MethodGet, uri.CreateResourceURI("", "environment/uuid", "", "", params), "application/json", nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var res types.UUIDResponse
+	err = json.Unmarshal(body, &res)
+	return &res, err
```

Comment:
shadow+scoping. 

Prefer `if err := nil {}`

Reply:
In this case, it's more succinct to not use the `if` statement, since I am returning both the result and error to the caller. Otherwise, yes, the if statement with err initialization makes more sense - where I don't use the error later.

Reply:
Shadowing err is a common habit in the go world, and its definitely a bad one.  A few extra characters has prevented weird bugs that are difficult to troubleshoot later on more times than I can count.

Reply:
Adjusted.